README.md

```diff
@@ -5,21 +5,27 @@ Kueue is a set of APIs and controller for [job](docs/concepts/workload.md)
 a job should be [admitted](docs/concepts#admission) to start (as in pods can be
 created) and when it should stop (as in active pods should be deleted).
 
+## Why use Kueue
+
+Kueue is a lean controller that you can install on top of a vanilla Kubernetes
+cluster without replacing any components. It is compatible with cloud
+environments where:
+- Nodes and other compute resources can be scaled up and down.
+- Compute resources are heterogeneous (in architecture, availability, price, etc.).
```

Comment:
```suggestion
Kueue is a lean controller that you can install on top of a vanilla Kubernetes
cluster. It does not replace any existing Kubernetes components. It is compatible with cloud
environments where:
- Compute resources are elastic and can be scaled up and down.
- Compute resources are heterogeneous (in architecture, availability, price, etc.).
```

Reply:
Done

Reply:
What does "heterogeneous" mean in this example?

Reply:
It is compatible, as homogeneous is just simpler. In kueue terms, if your resources are homegeneous, you need only one ResourceFlavor.